Client/Frontend/Settings/LoginDetailViewController.swift

```diff
@@ -256,11 +257,16 @@ extension LoginDetailViewController: UITableViewDelegate {
         }
 
         // Menu actions for Website
-        else if item == .WebsiteItem {
+        if item == .WebsiteItem {
             return action == MenuHelper.SelectorCopy || action == MenuHelper.SelectorOpenAndFill
-        } else {
-            return false
         }
+
+        // Menu actions for Username
+        if item == .WebsiteItem {
```

Comment:
I tried to understand how, given this bug, your newly added test still passed (logically it shouldn't as it should return `false` to `canPerformAction`. Turns out this delegate method is never being called during menu presentation.

I've tried to figure out why, but can't. It should be called immediately after `shouldShowMenuForRowAtIndexPath`. I can only think it's because you are doing stuff in the Notification observers for `UIMenuControllerWillShowMenuNotification`, although one would imagine that this notification wouldn't get fired at all if `canPerformAction` returns `false`.